src/util.js

```diff
@@ -51,3 +54,6 @@ export var notEmpty = function(o) {
 export var sanitise = function(str) {
     return typeof str === 'string' ? str.replace(/</g, '&lt;').replace(/>/g, '&gt;') : str;
 };
+export var mergeArray = function(arr) {
+    return (arr && arr.length ? arr.reduce((p, c) => p.concat(c)) : []);
```

Comment:
I think this can be implemented like the below, and maybe it's easier to understand.

```js
return Array.isArray(arr) ? [].concat(...arr) : [];
```

Reply:
done !